src/components/shared/ManagedTree.js

```diff
@@ -25,7 +25,7 @@ type Props = {
   onExpand?: (item: any) => void,
   onCollapse?: (item: any) => void,
   renderItem: any,
-  disabledFocus: boolean
+  disabledFocus?: boolean
```

Comment:
This was an attempt at making the prop optional to try and make the error mentioned in the PR
above go away.  The `disabledFocus` prop is not used in `src/components/PrimaryPanes/SourcesTree.js` so it should be optional for now.

Unfortunately making it optional did not make `yarn flow` happy.  I think this might be a known flow issue.